core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -123,7 +123,10 @@ export class WorkflowActionService {
   private readonly operatorGroup: OperatorGroup;
   private readonly syncTexeraModel: SyncTexeraModel;
   private readonly syncOperatorGroup: SyncOperatorGroup;
-
+  // variable to temporarily hold the current workflow to switch view to a particular version
+  private tempWorkflow = <Workflow>{};
+  // flag to disable workflow persist when displaying the read only particular version
+  private workflowPersistFlag = true;
```

Comment:
as mentioned, please consider move this to `WorkflowPersistService`.

Reply:
done